#//////////////////////////////////////////////////////////////////////
#//////////////////////////////////////////////////////////////////////
# ------------------ Fastfusion Parameters ----------------------------
#//////////////////////////////////////////////////////////////////////
#//////////////////////////////////////////////////////////////////////


# Desired Threading
threadFusion : true
threadMeshing : true

# Scales
imageScale : 1000.0
scale : 0.009
distanceThreshold : 0.03

# Mesh Saving (location + desired Name without ending)
saveMesh : false
fileLocation : /home/karrer/forPaper

# TF Id's
use_vicon_pose : true
world_id : /vicon
tracker_id : /camera_imu
cam_id : /realsense_frame

# Use time-window based reconstruction
# Set decay_time to value <= 0.0 to achieve full reconstruction,
# otherwise the defined time window (decay_time) is used
decay_time : 0.0

# Define if using Time of flight sensor or not
use_pmd : false

# Define if want to use Noise Data (only use with use_pmd = true)
depth_noise: true

# Want to use pcl visualizer (e.g. when performing global reconstruction)
use_pcl_visualizer : true
